### PR TITLE
Html file

### PR DESCRIPTION
**Pull request by joshuamc080**
made a website

### CODE REVIEW

**Comment by joshuamc080** [comment]
changed lmao hahahahahahaahahahahhaahhahaah

File: tests/Josh/joshgaysex.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+
+<h1>
+    Yo i kinda sad mans
```
Comment (ImmortalSul) [inline tests/Josh/joshgaysex.html:10]: Become happier man

**Review by joshuamc080 — COMMENTED** [review]
ok broman

File: tests/Josh/joshgaysex.html (hunk 1 above)
Comment (joshuamc080) [reply tests/Josh/joshgaysex.html:10]: ok lol

**Pull request merged**